package.json

```diff
@@ -10,6 +10,9 @@
   "scripts": {
     "postinstall": "husky install || true",
     "commit": "git-cz",
+    "test:format": "pnpm test:format:root && pnpm test:format:workspaces",
+    "test:format:root": "pnpm prettier --check '**/*.{js,cjs,mjs,ts,jsx,tsx,gql,graphql,graphqls,md,mdx,json}' --ignore-path='./.prettierignore'",
+    "test:format:workspaces": "pnpm --workspace-concurrency=1 -r exec prettier --check '**/*.{js,cjs,mjs,ts,jsx,tsx,gql,graphql,graphqls,md,mdx,json}' --ignore-path='./.gitignore'",
```

Comment:
Not that it really matters, but why restrict concurrency?

Reply:
Prettier output isn't recognized by pnpm 🤷 If running in parallel, the output is a mess. You have a list of file paths relative to the workspace roots in random order and there is no chance to guess which file belongs where - the workspaces paths are printed at the bottom.